swervelib/parser/json/DeviceJson.java

```diff
@@ -80,6 +80,7 @@ public SwerveAbsoluteEncoder createEncoder(SwerveMotor motor)
       case "dutycycle":
         return new PWMDutyCycleEncoderSwerve(id);
       case "thrifty":
+        return new AnalogAbsoluteEncoderSwerve(id);
```

Comment:
In java switch statements, when there are multiple case statements in a row like this, all those are combined into one.
Since the new return is exactly the same in both, this change should have no effect.
The error may be somewhere else in your code or setup.